contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseWindowPutOutputOperator.java

```diff
@@ -46,8 +40,6 @@
  * the tuple in the table.<br>
  *
  * <br>
- * This class provides a batch put where tuples are collected till the end
- * window and they are put on end window
  *
```

Comment:
All same comments for `AbstractHBaseWindowAppendOutputOperator` apply here as well.